src/marionette.sliding-view.js

```diff
@@ -75,8 +80,9 @@ Mn.SlidingView = Mn.CollectionView.extend({
       }
     }
 
-    // If the boundaries are unchanged, then we bail out early
-    if (lowerBound === this._lowerBound && upperBound === this._upperBound) {
+    // If the boundaries are unchanged and we're not forcing updates then abort the update
+    var boundsUpdated = lowerBound === this._lowerBound && upperBound === this._upperBound;
```

Comment:
This variable name doesn't match what it's checking. If it's `true`, then the bounds *haven't* updated.